### DISCUSSION TITLE
Page layout may be unexpected due to Quirks Mode

### DISCUSSION THREAD

**Discussion by VMLuca** [vercel/next.js#66070] [App Router]
I've come across this warning and I've done some research to find out that it's a history feature for browsers. I've also read that next.js automatically adds the <!DOCTYPE html> attribute, which should solve this issue. I can confirm that next.js does this and adds it, but I'm still wondering why I'm getting this warning.

**Comment by icyJoseph** [comment]
Aha, interesting, does your HTML start with doctype?

**Comment by VMLuca** [comment]
![image](https://github.com/vercel/next.js/assets/149671903/e3e406f1-f2af-4843-b23b-788cb285c1fe)
Yes, it does.

**Reply by icyJoseph** [reply]
🤔 where do you see the warning?

**Reply by VMLuca** [reply]
![image](https://github.com/vercel/next.js/assets/149671903/03f0da85-2559-4be8-a7b6-5ed3a2c78029)
In the issues tab of chrome but magically it does not appear any more 😅

**Reply by icyJoseph** [reply]
Did you add/remove any chrome extensions lately?

**Reply by VMLuca** [reply]
No, I haven't changed any Chrome extensions. Thanks for your quick response. I'll keep an eye on it to see when it occurs again. Thanks, mate!
